k8s/upgrade/src/bin/upgrade-job/helm/upgrade.rs

```diff
@@ -178,6 +193,15 @@ impl HelmUpgraderBuilder {
 
         // Determine chart variant.
         if Regex::new(umbrella_regex.as_str())?.is_match(chart.as_ref()) {
+            // The version of the Core helm chart (installed as a dependent chart) which is
+            // installed in the cluster.
+            let source_version =
+                core_version_from_umbrella_release(&client, helm_release.name()).await?;
+            debug!(version=%source_version, "Found version of dependent chart {CORE_CHART_NAME}");
```

Comment:
These would be useful as info?

```suggestion
            info!(version=%source_version, "Found version of dependent chart {CORE_CHART_NAME}");
```